addons/skin.estuary/1080i/Home.xml

```diff
@@ -565,6 +565,20 @@
 							<param name="item6_label" value="$LOCALIZE[137]" />
 							<param name="item7_vis" value="False" />
 							<param name="item8_vis" value="False" />
+							<param name="visible" value="System.HasPVRAddon"/>
+						</include>
+						<include content="ImageWidgetHeader">
+							<param name="widget_header" value="$LOCALIZE[19020]" />
+							<param name="visible" value="!System.HasPVRAddon"/>
+						</include>
+						<include content="ImageWidget">
+							<param name="image_path" value="special://skin/extras/home-images/tv.jpg"/>
+							<param name="text_label" value="$LOCALIZE[31143]" />
+							<param name="button_label" value="$LOCALIZE[31144]" />
+							<param name="button_onclick" value="ActivateWindow(addonbrowser,addons://,return)"/>
```

Comment:
we are .. use `ActivateWindow(addonbrowser,addons://user/xbmc.pvrclient,return)`